packages/frontend/web/components/Header/Nav/ReaderRevenueLinks.tsx

```diff
@@ -120,58 +97,66 @@ export const ReaderRevenueLinks: React.FC<{
         contribute: string;
     };
     dataLinkNamePrefix: string;
-}> = ({ edition, urls, dataLinkNamePrefix }) => {
+    noResponsive: boolean;
```

Comment:
It might be worth splitting these buttons out slightly, the markup got slightly complex with the `noResponsive`?

Reply:
Yeah, this is a case where a design system would help as my hesitancy to separate was the level of refactoring to avoid duplication here. I'll have a look.

Reply:
Made a `RRbutton` for clarity